tools/sambamba/sambamba.xml

```diff
@@ -0,0 +1,222 @@
+<tool id="sambamba" name="sambamba" version="">
```

Comment:
You need to have a version. Usually the same version as the tool with a `.0` at the end.